### PR TITLE
fix package imports for pilot

### PR DESCRIPTION
**Pull request by kuafou**

### CODE REVIEW

File: pilot/pkg/serviceregistry/cloudfoundry/serviceaccounts.go [hunk 1]
```diff
@@ -14,10 +14,11 @@
 
 package cloudfoundry
 
-import "istio.io/istio/pilot/pkg/model"
```
Comment (venilnoronha) [inline pilot/pkg/serviceregistry/cloudfoundry/serviceaccounts.go]: nit: this could be kept as is.

Comment (kuafou) [reply pilot/pkg/serviceregistry/cloudfoundry/serviceaccounts.go]: cloud i resolve this ?

Comment (venilnoronha) [reply pilot/pkg/serviceregistry/cloudfoundry/serviceaccounts.go]: Yes, I generally refer the commit by ID when resolving comments.

**Comment by venilnoronha** [comment]
This is mostly because of merging `master` onto your branch. Could you try to re-work this PR without merging `master`? You could rebase with `master` if needed, and I'm hopeful that the CLA issue will get resolved with that.

**Comment by kuafou** [comment]
@yangminzhu This caused by `git rebase upstream/master` and may i close this pr then create a new one ?

**Comment by venilnoronha** [comment]
Yes, if that helps.

**Comment by kuafou** [comment]
> This is mostly because of merging `master` onto your branch. Could you try to re-work this PR without merging `master`? You could rebase with `master` if needed, and I'm hopeful that the CLA issue will get resolved with that.

Thanks a lot.

**Pull request closed without merging**